readme.md

```diff
@@ -0,0 +1,44 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
+
+> My first project at microverse with a simple message "Hello Microverse"
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/3nityGitHub)
+- Twitter: [@twitterhandle](https://twitter.com/tbarijohnson)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] You have added a license link here but you forgot to add the license file  :fearful: No worries you can add it using these [guildlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: